automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/api/sam/SamApiSpec.scala

```diff
@@ -24,11 +24,15 @@ class SamApiSpec extends FreeSpec with Matchers {
     Await.result(remove, 5.seconds)
   }
 
-  def findPetInGoogle(userInfo: UserStatusDetails): Option[WorkbenchUserServiceAccount] = {
-    val find = googleIamDAO.findServiceAccount(GoogleProject(Config.Projects.default), petName(userInfo))
+  def findSaInGoogle(name: WorkbenchUserServiceAccountName): Option[WorkbenchUserServiceAccount] = {
+    val find = googleIamDAO.findServiceAccount(GoogleProject(Config.Projects.default), name)
     Await.result(find, 5.seconds)
```

Comment:
Why are we cutting it off at 5 seconds? Are there times the Future just hangs?

Reply:
shrug-emoji we have to pick some value

Reply:
and 1 sec is too short